src/pages/en/tutorial/0-introduction/1.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: ~/layouts/TutorialLayout.astro
+title: Background Knowledge
+setup: |
+  import Badge from '~/components/Badge.astro';
+  import Checklist from '~/components/Checklist.astro';
+  import Goals from '~/components/tutorial/Goals.astro';
+---
+
+<Goals>
+  - learned what background programming knowledge you will need to get started building with Astro
+  - chosen and installed any tools that you will use to build your Astro website
+  - set up the necessary online accounts to deploy your site to the web
+</Goals>
```

Comment:
I'm unsure about benefits of showing learning objectives to the user (I love them to structure writing) especially for a small section. I can definitely be convinced, flagging as worth talking about

Reply:
We talked about this. They stay for now. (Will strive to make the others friendlier)